test/config/utility.cc

```diff
@@ -0,0 +1,99 @@
+#include "test/config/utility.h"
+
+#include "common/common/assert.h"
+#include "common/protobuf/utility.h"
+
+#include "test/test_common/environment.h"
+#include "test/test_common/network_utility.h"
+
+#include "api/filter/http_connection_manager.pb.h"
+
+namespace Envoy {
+
+const std::string basic_config = R"EOF(
+static_resources:
+  listeners:
+    filter_chains:
+      filters:
+        name: http_connection_manager
+        config:
+          http_filters:
+            name: router
+            config:
+              deprecated_v1: true
+          codec_type: HTTP1
+          route_config:
+            virtual_hosts:
+              name: integration
+              routes:
+                route:
+                  cluster: cluster_0
+                match:
+                  prefix: "/"
+              domains: "*"
+            name: route_config_0
+  clusters:
+    name: cluster_0
+    connect_timeout: 5s
+    hosts:
+      socket_address:
+        address: 127.0.0.1
+        port_value: 0
+admin:
+  access_log_path: /dev/null
+  address:
+    socket_address:
+      address: 127.0.0.1
+      port_value: 0
+)EOF";
+
+ConfigHelper::ConfigHelper(const Network::Address::IpVersion version) {
+  // Add the default listener.
+  addListener("listener_0", version);
```

Comment:
I'm a little confused what this is doing. Don't we want this to specify a filter stack somehow and/or some other listener config template?

Reply:
Oh, I left it pulling the filter stack from the basic config.

How about I roll this bit back, and we can refine and add helpers as we port tests over.